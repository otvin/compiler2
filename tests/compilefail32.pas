program compilefail32(output);
{cannot assign to a constant}

const two=2;

procedure addtwo(a:integer);
const localtwo = two;
begin
	writeln(a + localtwo);
end;

begin
	two := 2;
	addtwo(17);
end.
