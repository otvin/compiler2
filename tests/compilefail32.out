Cannot assign to constant 'two' in file:tests/compilefail32.pas line:13    column:5  
